packages/rds/src/styles/default.scss

```diff
@@ -0,0 +1,99 @@
+/**
+ * Default CSS styles. This file is similar to a "reset" stylesheet and
+ * represents base styles for consistency across all RDS components.
+ *
+ * This stylesheet can be imported in two different ways:
+ *
+ * 1) Import within `BaseLayout.tsx`:
+ *
+ * ```
+ * import '@blinkk/rds/styles/default.scss';
+ * ```
+ *
+ * Note: Cannot configure the styles using this method. See (2) if you need to
+ * configure the CSS output.
+ *
+ * 2) Import within a SCSS file, e.g. `global.scss`:
+ *
+ * ```
+ * @use '@blinkk/rds/styles/default.scss';
+ * ```
+ *
+ * To change any configurations, use the `@use ... with` syntax, e.g.:
+ *
+ * ```
+ * @use '@blinkk/rds/styles/default.scss' with (
+ *   $disable-prefers-reduced-motion: true;
+ * );
+ * ```
+ */
+
+$disable-smooth-scroll: false !default;
+$disable-prefers-reduced-motion: false !default;
+
+@layer rds.base {
```

Comment:
@layer is relatively new to the team.

If we want to override any of the rds.base layer, I think we would also need to wrap it in the project level layer otherwise it won't win the specificity battle. 

Maybe we should add an example of that?

```
@use '@blinkk/rds/styles/default.scss';
@layer my-project {
   img {
     max-width: none;
   }
}
```

Reply:
When no cascade layer is set, it wins the specificity battle over all other cascade layers. Doing this would be enough to override the rds.base layer:

```
img {
  max-width: auto;
}
```

Reply:
Ah gotcha.  I'm still picking up my knowledge on layers.  Thanks for confirming :)